src/components/TeacherComponents/CopyrightNotice/CopyrightNotice.tsx

```diff
@@ -10,6 +10,7 @@ import Icon from "@/components/SharedComponents/Icon";
 type CopyrightNoticeProps = FontProps & {
   showPostAlbCopyright: boolean;
   openLinksExternally: boolean;
+  updatedAt?: string;
```

Comment:
should this be optional?

Reply:
aha, in that case it might better to have this component accept a copyrightYear param, and have the logic higher up? because it's not clear here that this is handling that one specific edge case

Reply:
So this component is also being utilised by Pupils and Curriculum squad, which is another reason why I think I made it optional as it would mean changing their MVs, would it still be worth refactoring?

Reply:
I think it would be worth raising that in the cross squad channel, presumably they also need to show the correct copyright year? 
personally I would suggest we update their components so that for now they just pass a hardcoded year and they can update later when it's a priority for them